### PR TITLE
✨(User): Add user data to redux.

### PR DESCRIPTION
**Pull request by jansen0825**
# Description
- Fetches user data and stores it into a reducer state property.
- This is a pre-work for auto assigning owners to a story and other features. 

Screen Shot 2020-10-26 at 1 30 55 AM

### CODE REVIEW

File: handlers/PivotalHandler.ts [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
-import { Project, Story } from '../redux/types';
+import { Project, Story, User } from '../redux/types';
 
 const PIVOTAL_API_URL = 'https://www.pivotaltracker.com/services/v5';
 
 const STORY_FIELDS =
   'fields=name,estimate,owners,labels,blockers,reviews,story_type,description,comments(id,person,text,created_at),current_state';
 
 class PivotalHandler {
+  static async getUser({ apiKey }): Promise<User> {
```
Comment (mattvv) [inline handlers/PivotalHandler.ts:12]: clean!

**Review by mattvv — APPROVED** [review]

**Comment by mattvv** [comment]
Given that we get users projects from this? maybe we could phase our the getProjects call?

**Comment by mattvv** [comment]
@jansen0825 I looked into making it one request and it turns out I can't get a lot of subscope items I will need from the project from that /me endpoint so i'm leaving at two requests. We should probably consolodate all the initial stuff we load at the start of a page at some point instead of lazy loading, but for now this is fine. Feel free to merge

**Review by jansen0825 — COMMENTED** [review]
Sounds great sir! Thanks for jumping into this.

**Pull request merged**